core/controllers/reader.py

```diff
@@ -721,16 +721,59 @@ class ReaderFeedbackHandler(base.BaseHandler):
 
     REQUIRE_PAYLOAD_CSRF_CHECK = False
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.ENTITY_ID_REGEX
+                }]
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'subject': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'feedback': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'has_length_at_most',
+                        'max_value': 10000
+                    }]
+                },
+                'default_value': None
+            },
+            'include_author': {
+                'schema': {
+                    'type': 'bool'
+                }
+            },
+            'state_name': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            }
```

Comment:
Any rules for these?

Reply:
@vojtechjelinek I added rules for 'state_name'. Should I add more rules for other arguments?